databricks/koalas/tests/test_groupby.py

```diff
@@ -494,6 +494,84 @@ def test_aggregate_relabel(self):
         )
         self.assert_eq(agg_kdf, agg_pdf)
 
+    def test_dropna(self):
+        # pd.DataFrame.groupby with dropna parameter is implemented since pandas 1.1.0
+        if LooseVersion(pd.__version__) >= LooseVersion("1.1.0"):
+            pdf = pd.DataFrame(
+                {"A": [None, 1, None, 1, 2], "B": [1, 2, 3, None, None], "C": [4, 5, 6, 7, None],}
+            )
+            kdf = ks.from_pandas(pdf)
+
+            for dropna in [True, False]:
+                for as_index in [True, False]:
+                    if as_index:
+                        sort = lambda df: df.sort_index()
+                    else:
+                        sort = lambda df: df.sort_values("A").reset_index(drop=True)
+
+                    self.assert_eq(
+                        sort(kdf.groupby("A", as_index=as_index, dropna=dropna).std()),
+                        sort(pdf.groupby("A", as_index=as_index, dropna=dropna).std()),
+                    )
+
+                    self.assert_eq(
+                        sort(kdf.groupby("A", as_index=as_index, dropna=dropna).B.std()),
+                        sort(pdf.groupby("A", as_index=as_index, dropna=dropna).B.std()),
+                    )
+                    self.assert_eq(
+                        sort(kdf.groupby("A", as_index=as_index, dropna=dropna)["B"].std()),
+                        sort(pdf.groupby("A", as_index=as_index, dropna=dropna)["B"].std()),
+                    )
+
+                    for aggfunc in ["max", "std"]:
+
+                        sorted_agg_kdf = sort(
+                            kdf.groupby("A", as_index=as_index, dropna=dropna).agg(aggfunc)
+                        )
+                        sorted_agg_pdf = sort(
+                            pdf.groupby("A", as_index=as_index, dropna=dropna).agg(aggfunc)
+                        )
+                        self.assert_eq(sorted_agg_pdf, sorted_agg_kdf)
+
+            for aggfunc in ["max", "std"]:
+                for dropna in [True, False]:
+                    for as_index in [True, False]:
+                        if as_index:
+                            sort = lambda df: df.sort_index()
+                        else:
+                            sort = lambda df: df.sort_values(["A", "B"]).reset_index(drop=True)
+
+                        sorted_agg_kdf = sort(
+                            kdf.groupby(["A", "B"], as_index=as_index, dropna=dropna).agg(aggfunc)
+                        )
+                        sorted_agg_pdf = sort(
+                            pdf.groupby(["A", "B"], as_index=as_index, dropna=dropna).agg(aggfunc)
+                        )
+                        self.assert_eq(sorted_agg_pdf, sorted_agg_kdf, almost=True)
```

Comment:
And this can be

```python
for dropna in [True, False]:
    for as_index in [True, False]:
        if as_index:
            sort = lambda df: df.sort_index()
        else:
            sort = lambda df: df.sort_values(["A", "B"]).reset_index(drop=True)

        self.assert_eq(
            sort(kdf.groupby(["A", "B"], as_index=as_index, dropna=dropna).agg({"C": ["min", "std"]})),
            sort(pdf.groupby(["A", "B"], as_index=as_index, dropna=dropna).agg({"C": ["min", "std"]})),
            almost=True,
        )
```